src/pages/workspaces/workspace/analysis/modals/modalStyles.ts

```diff
@@ -0,0 +1,12 @@
+import colors from 'src/libs/colors'
+
+//I re-create compute styles here because they're not typed in their native file
+export const computeStyles: Record<string, React.CSSProperties> = {
+  label: { fontWeight: 600, whiteSpace: 'pre' },
+  value: { fontWeight: 400, whiteSpace: 'pre' },
+  titleBar: { marginBottom: '1rem' },
+  whiteBoxContainer: { padding: '1rem', borderRadius: 3, backgroundColor: 'white' },
+  drawerContent: { display: 'flex', flexDirection: 'column', flex: 1, padding: '1.5rem' },
+  headerText: { fontSize: 16, fontWeight: 600 },
+  warningView: { backgroundColor: colors.warning(0.1) }
+}
```

Comment:
Would love to hear thoughts from another reviewer on this file.
My thoughts: I don't think `computeStyles` should live in `runtime-utils.js`. Additionally, we need types on `computeStyles` for this PR, so I suggested we put it in it's own file.
Is this type of file appropriate? Or is there a better convention we should be following for shared style objects?

Reply:
Hmm... I'm ok with it having its own file, seems appropriate since its used by a lot of components

Reply:
I like the separate styles file. Leaves the "utils" files for more business logic.